playground/src/aztecEnv.ts

```diff
@@ -30,10 +32,23 @@ export class WebLogger {
   private static instance: WebLogger;
   private logs: Log[] = [];
 
-  private constructor(private setLogs: (logs: Log[]) => void) {}
+  private static updateIntervalMs = 1000;
+  private static readonly MAX_LOGS_TO_KEEP = 200;
+
+  private pendingUpdate = true;
```

Comment:
This seems unused?

Reply:
My vscode also marks it as unused, but it is definitely used 🙃

Reply:
No I mean, we only ever set this value, but we never read it

Reply:
FFS I'm stupid, forgot about it from a test